ecommers/lib/ui/widgets/search/search_text_filed.dart

```diff
@@ -0,0 +1,46 @@
+import 'package:ecommers/generated/i18n.dart';
+import 'package:ecommers/ui/decorations/dimens/index.dart';
+import 'package:ecommers/ui/decorations/index.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/widgets.dart';
+
+class SearchTextFiled extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Row(
+      children: <Widget>[
+        Expanded(
+          child: Container(
+            height: 35,
+            decoration: BoxDecoration(
+              color: BrandingColors.blur,
+              borderRadius: const BorderRadius.all(
+                Radius.circular(20),
```

Comment:
The correct radius for searchField is 18.
Add please property with 18 value to the Radiuses class and use it here